FAQ.md

```diff
@@ -255,6 +256,13 @@ If you just ran `dep init`, there should be a copy of your original vendor direc
 > dep assumes complete control of vendor/, and may indeed blow things away if it feels like it.
 -[@peterbourgon in #206](https://github.com/golang/dep/issues/206#issuecomment-277139419)
 
+## How do I make `dep` resolve dependencies from my `GOPATH`?
+By default, `dep` runs in network mode and would download all the dependencies
+from the network while initializing (`init`) and restoring (`ensure`). `init`
+provides an option to scan the `GOPATH` for dependencies by doing
+`dep init -gopath`, which falls back to network mode when the packages are not
```

Comment:
> falls back to

We actually have to rely on a solve (which likely hits the network) no matter what - otherwise there's no guarantee that what's on GOPATH is actually a satisfiable answer. 😢